### PR TITLE
Update "What is OpenCensus"

### PR DESCRIPTION
**Pull request by ruthnaebeck**

### CODE REVIEW

**Comment by ruthnaebeck** [comment]
I signed it!

**Comment by mtwo** [comment]
Shoot, sorry for the delay. Taking a look now!

File: content/_index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ class: "no-pagination no-top-border-header no-search max-text-width"
 
 ##### What is OpenCensus?
 
-{{<title>}} is a single distribution of libraries that collect **metrics** and **distributed traces** from your services
+{{<title>}} allows you to collect application **metrics** and **distributed traces**, then transfer the data to a backend of your choice in real time. This data can be analyzed by developers and admins to understand the health of the application and debug problems.
```
Comment (mtwo) [inline content/_index.md]:
```suggestion
{{<title>}} is a set of libraries for various languages that allow you to collect application **metrics** and **distributed traces**, then transfer the data to a backend of your choice in real time. This data can be analyzed by developers and admins to understand the health of the application and debug problems.
```

**Comment by songy23** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. 

This is due to commits from multiple authors. Manually verified.

**Pull request merged**